### ISSUE TITLE
Adding the ability to launch local and aws-sbatch executors within the same nf script

### ISSUE THREAD

**Issue by srynobio** [nextflow-io/nextflow#631]
`[Feature Request]`

Adding the ability to mix both (or more) of these executor types within the same script would be a nice feature to have included.  Currently it requires you to create individual nf script. 

* Included conversation from Gitter:

```
@pditommaso I wanted to get your thoughts on mixing aws-abatch and local executors within the same nf script. I did some limited testing and it seem that they can not be mixed, mainly due to the inclusion of -w s3://bucket to the command line, i.e. the local tasks try to use the s3 work space and fail.
```

**Comment by rohanshah** [comment]
I reached out Gitter about this as well, here is a sort of high-level view of why this might be useful for us:

We have some small processes that need to interact with AWS services to small setup work for other processes outside of Nextflow that run quickly and can happen in parallel with the ongoing, larger workflow. It would be helpful to execute these small processes locally and have the rest of the workflow continue to run in AWS Batch

**Comment by pditommaso** [comment]
This was a duplicated of #744. This feature is implemented by in version 18.10.1. See [here](https://www.nextflow.io/docs/latest/awscloud.html#hybrid-workloads).
